charter.adoc

```diff
@@ -15,20 +15,20 @@ The Toolchains SIG develops the strategy, identifies gaps, and prioritizes tasks
 
 The SIG's main goals are:
 
-- Push RISC-V into open-source software
-- Pull developers to the RISC-V ecosystem
+- Push RISC-V into open-source toolchain projects
+- Pull developers to the RISC-V toolchain ecosystem
 - Advice on ISA extensions
 
 To achieve these goals, the following steps are taken:
 
-- Advice on ISA extensions and ensure that all future ISA extensions are driven by demonstrable/quantifiable benefits and possibly support them in development tools. Enable the SW HC to drive the ICs to develop extensions necessary to support the strategy.
+- Promote RISC-V as an ISA to be used for research into novel compilation techniques.
+- Track the support for RISC-V across development tools and identify gaps.
 - Identify and work with the key open-source projects for the RISC-V software ecosystem: compilers, debuggers, performance analysis tools, libraries implementing performance primitives, and memory management libraries.
-- Liaise between open-source compiler developers and RISC-V task groups defining ISA, profiling, and debug extensions.
+- Liaise between open-source compiler developers and RISC-V groups defining new extensions.
+- Streamline RISC-V-specific ABIs (e.g. C calling convention), APIs (e.g. intrinsics, attributes), ASM language, and other interfaces (e.g. command line flags) in the RISC-V toolchain ecosystem.
+- Advice on ISA extensions and ensure that all future ISA extensions are driven by demonstrable/quantifiable benefits and possibly support them in development tools.
 - Reach out to developers and decision-makers to contribute to the RISC-V ecosystem.
-- Promote RISC-V as an ISA to be used for research into novel compilation techniques.
-- Track the support for RISC-V across development tools.
 - Support benchmarking and work with benchmark organizations.
-- Benchmark development tools for RISC-V and provide reporting to internal RISC-V committees on the relative progress of development tools targeting RISC-V.
 - Provide access to state-of-the-art development tools and compilation techniques for all RISC-V task groups by fostering an environment of cooperation between academia, open-source tool development, and the RISC-V ecosystem.
 - Engage with new programming languages, programming paradigms, and development tool development towards the strategic goal of making RISC-V the prime choice ("First on RISC-V") for innovation in development tools.
 
```

Comment:
We could also add: 
- Promote support for embedded features in the toolchains (e.g., debuggability of linker scripts) since embedded domain is a great application area for RISC-V.